pkg/inspect/writer.go

```
@@ -0,0 +1,61 @@
+
+func writeYAML(filepath string, obj runtime.Object) error {
+	dir := path.Dir(filepath)
+	if err := os.MkdirAll(dir, folderPermission); err != nil {
+		return fmt.Errorf("unable to create dir %s: %w", dir, err)
+	}
+
+	f, err := os.OpenFile(filepath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, filePermission)
+	if err != nil {
+		return fmt.Errorf("unable to create file %s: %w", filepath, err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)